### DISCUSSION TITLE
QField packaging issue

### DISCUSSION THREAD

**Discussion by r1974-tt** [opengisch/QField#5440] [Q&A]
Hello to the community
I'm having a problem using Qfield for field data entry projects. I have tried several compatible combinations of Qgis and QFieldSync versions. I work on Windows and Android.
In the future I'd like to be able to synchronise data from several mobile data collection devices on my computer by cable (free use with no limit on the amount of data). 

1/ I create a "TEST.qgs project" in a local folder TEST QFIELD/projet TEST
2/ I create an Offline Editing points layer in a "GPKG TEST.gpkg|layer name = WPTS" geopackage
3/ I use QFieldSync to package my project and my layer in an export subfolder TEST QFIELD/projet TEST/Export
4/ I copy/paste this sub-folder into my phone 
5/ I create entities on my mobile phone
6/ I copy/paste from my phone all the files in the folder containing the project used for data collection to an import folder on my computer: QFIELD TEST/projet TEST/Import 
6/ I open my local project created in step 1: TEST QFIELD/projet TEST/projet TEST.qgs
7/ I use QFieldSync to synchronise the data collected in the field, which should be located in the folder created in step 6, to this "TEST.qgs project".

And then... nothing happens! The WPTS layer of the GPKG TEST.gpkg does not contain any entity previously created on the mobile device ... !

I have the suspicion that the problem comes from the fact that when I package my project on my computer in step 3, QfieldSync does not create a copy of my offline editable layer, whether it is saved in shp format or directly in geopackage :

![Probleme Qfield_Image1](https://github.com/user-attachments/assets/b7441cf8-c966-4e72-8228-d25233debf77)

 
Also when I edit entities in Qfield on my phone, the modification date on my phone in the Export/data.gpkg or Export/project TEST_qfield.qgs files is never updated to match the date I created entities in QField on my phone.
This gives me the impression that on my phone, Qfield doesn't know/can't find where to write changes (entity additions) as the Export folder doesn't contain a copy of my offline editable layer. 

To overcome this problem, I tried copying/pasting the layers directly from my computer to my phone without using QFieldSync as recommended on the forums I consulted: 
https://github.com/opengisch/QField/discussions/3914

![Probleme Qfield_Image2](https://github.com/user-attachments/assets/80740c90-c129-4053-9bbe-6c5bdb96c087)

 
With this procedure, the offline editable layer is "copied and pasted" manually on the phone into the project file folder (last image step 1). But here, I obviously can't 'synchronise' the data from several phones to a computer, but only copy/paste the layers edited by each mobile device (last image step 3) and then I have to merge all these entities... This isn't a smooth enough procedure for frequent work by several people on the same layer...

Could someone please help me identify my packaging and/or synchronisation problem (I'd like to make it clear once again that I don't want to use QFieldCloud in the future)?

Many thanks!

**Comment by SeqLaz** [comment]
Hello @r1974-tt Could you please provide a sample of one project before packaging and another project already packaged via QFieldSync?
